x/dex/client/cli/query_limit_order_tranche_user.go

```diff
@@ -45,18 +45,23 @@ func CmdListLimitOrderTrancheUser() *cobra.Command {
 
 func CmdShowLimitOrderTrancheUser() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "show-limit-order-tranche-user [address] [tranche-key]",
+		Use:     "show-limit-order-tranche-user [address] [tranche-key] ?(--calc-withdraw)",
 		Short:   "shows a LimitOrderTrancheUser",
-		Example: "show-limit-order-tranche-user TRANCHEKEY123 alice",
+		Example: "show-limit-order-tranche-user alice TRANCHEKEY123",
```

Comment:
I think it's better to use valid neutron address as an example instead of `alice`